policyengine_us/parameters/gov/irs/uprating.py

```diff
@@ -15,12 +15,23 @@ def get_irs_cpi(parameters: ParameterNode, year: int) -> float:
 def set_irs_uprating_parameter(parameters: ParameterNode) -> ParameterNode:
 
     IRS_UPRATING_START_YEAR = 2010
-    IRS_UPRATING_END_YEAR = 2035
+    IRS_UPRATING_END_YEAR = 2100
 
     uprating_index: Parameter = parameters.gov.irs.uprating
     # Note: range is inclusive of start and exclusive of end, so we add 1
     for year in range(IRS_UPRATING_START_YEAR, IRS_UPRATING_END_YEAR + 1):
-        irs_cpi = get_irs_cpi(parameters, year - 1)
+        if year <= 2035:
```

Comment:
If there a way to avoid hard coding the years? Might leave room for oversight if we need to update annually